runtime/onert/backend/cpu/KernelGenerator.cc

```diff
@@ -480,7 +480,8 @@ void KernelGenerator::visit(const ir::operation::Reshape &node)
 
   auto fn = std::make_unique<ops::ReshapeLayer>();
 
-  fn->configure(input_tensor, shape_tensor, output_tensor);
+  const auto input_consumers_number = _ctx.at(input_index).getUses().size();
```

Comment:
What about using the same name `uses` instead of introducing another name `consumers`?
```suggestion
  const auto input_uses_number = _ctx.at(input_index).getUses().size();
```